universal/pm.md

```diff
@@ -43,7 +51,25 @@ XCPM does not natively support Haswell-E and Broadwell-E, this means we need to
     * Cpuid1Data: `D4060300 00000000 00000000 00000000`
     * Cpuid1Mask: `FFFFFFFF 00000000 00000000 00000000`
 
-## Using CPU Friend
+:::
+
+## Manually Modifying Power Management Data
+
+In most cases, the native CPU power management data shipped with macOS work out of the box. If you're experiencing issues, changing your SMBIOS to something more appropriate to your system will provide different data and may be better for your usecase. In the cases where manual tuning is required, you can use CPUFriend to inject modified power management data, but if you don't know what you're doing, you can severely break power management.
+
+::: warning
+
+In most cases, you do not have to do this. Change your SMBIOS instead.
+
+:::
+
+::: tip
+
+This is an example on how to change some parts of power management data. For more information, you should check out [CPUFriend's documentation](https://github.com/acidanthera/CPUFriend/blob/master/Instructions.md).
+
+:::
+
+### Using CPUFriend
 
 To start, we're gonna need a couple things:
 
```

Comment:
For the line below:

`X86PlatformPlugin` is also used on Ivy Bridge, just with AICPUPM. Should be Sandy Bridge instead I think?